File: 0392-is-subsequence/0392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        
        if(s.empty())return true;
        int i=0;
        int j=0;
        while(i<t.length() && j<s.length()){
            if(s[j]==t[i]){
                j++;
            }
            i++;
        }
        return (j==s.length());
    }
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if(len(s)==0):
            return True
        i,j=0,0
        for i in range(0,len(t)):
            if(s[j]==t[i]):
                j+=1;
            else:
                continue
            if(j==len(s)):
                return True
        return False